pyproject.toml

```diff
@@ -24,6 +26,9 @@ pytest-cov = "^4.0.0"
 pytest-runner = "^6.0.0"
 pytest-watch = "^4.2.0"
 
+[tool.poetry.group.dev.dependencies]
+jupyter = "^1.0.0"
```

Comment:
@billfreeman44 We need to use either `tool.poetry.dev-dependencies` or `tool.poetry.group.dev.dependencies` but not both.